docs/api/features/manifold.rst

```diff
@@ -142,11 +150,32 @@ continuous by counting the number of unique values in ``y``.
 
 .. image:: images/concrete_isomap_manifold.png
 
+
+Quick Method
+-----------------
+The same functionality above can be achieved with the associated quick method ``manifold_embedding``. This method will build the ``Manifold`` object with the associated arguments, fit it, then (optionally) immediately show the visualization.
+
+.. note to contributors: the below code takes a long time to run so has not been
+   modified with a plot directive. See manifold.py to regenerate images.
+
+.. code:: python
+
+    from yellowbrick.features.Manifold import manifold_embedding
+    from yellowbrick.datasets import load_concrete
+
+    # Load the regression dataset
+    X, y = load_concrete()
+
+    # Instantiate the visualizer
+    manifold_embedding(X, y, manifold="isomap", n_neighbors=10)
+
+.. image:: images/concrete_quick_manifold.png
```

Comment:
We need to write code for producing this image in `docs/api/ features/manifold.py`.

Reply:
Yes, @naresh-bachwani I have made the changes to the ### docs/api/features/manifold.py.